Notes app

Ayush-Dahala opened the pull request:
# Zero To Mastery - Webblocks 2022 📦📦

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- In this project you can take your daily notes and delete them as you wanted and search them based on keyword

## Breaking changes
-none

## Screenshots
![image](https://user-images.githubusercontent.com/101979643/194748344-ef3d6372-8d24-46bf-82bd-8fcf7dcac1a5.png)

sophiabrandt commented on src/blocks/Ayush-Dahala/index.js:
```diff
@@ -0,0 +1,83 @@
+console.log("Welcome to my notes")
```
Please remove `console.log()` from production code.

sophiabrandt reviewed (changes requested):
Please remove `console.log`.

Ayush-Dahala commented:
Updated I removed all the console.log

On Mon, 10 Oct, 2022, 2:13 pm Sophia Brandt, ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In src/blocks/Ayush-Dahala/index.js
> <https://github.com/zero-to-mastery/webblocks-2022/pull/163#discussion_r991040665>
> :
>
> > @@ -0,0 +1,83 @@
> +console.log("Welcome to my notes")
>
> Please remove console.log() from production code.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zero-to-mastery/webblocks-2022/pull/163#pullrequestreview-1135662430>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYKBL6ZIH2MSFZKH6Z6I6KLWCPJKTANCNFSM6AAAAAARATSXGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Ayush-Dahala commented:
> Please remove `console.log`.

yes I removed all the console.log

MattCSmith reviewed (approved):
Looks good to me 👍
